homeassistant/components/utility_meter/sensor.py

```diff
@@ -48,22 +48,24 @@
         conf_meter_source = hass.data[DATA_UTILITY][meter][CONF_SOURCE_SENSOR]
         conf_meter_type = hass.data[DATA_UTILITY][meter].get(CONF_METER_TYPE)
         conf_meter_offset = hass.data[DATA_UTILITY][meter][CONF_METER_OFFSET]
+        conf_meter_net_consumption =\
+            hass.data[DATA_UTILITY][meter][CONF_METER_NET_CONSUMPTION]
         conf_meter_tariff_entity = hass.data[DATA_UTILITY][meter].get(
             CONF_TARIFF_ENTITY)
 
         meters.append(UtilityMeterSensor(
             conf_meter_source, conf.get(CONF_NAME), conf_meter_type,
-            conf_meter_offset, conf.get(CONF_TARIFF),
-            conf_meter_tariff_entity))
+            conf_meter_offset, conf_meter_net_consumption,
+            conf.get(CONF_TARIFF), conf_meter_tariff_entity))
 
     async_add_entities(meters)
 
 
 class UtilityMeterSensor(RestoreEntity):
     """Representation of an utility meter sensor."""
 
-    def __init__(self, source_entity, name, meter_type, meter_offset=0,
-                 tariff=None, tariff_entity=None):
+    def __init__(self, source_entity, name, meter_type, meter_offset,
+                 net_consumption, tariff=None, tariff_entity=None):
```

Comment:
I would leave the default value in
```
meter_offset=0, net_consumption=False
```

Reply:
@fabaff mentioned that it was already set as a default and the way the code is setup it should never use that default anyway.

Reply:
👍